src/NServiceBus.AcceptanceTests/NonDTC/When_sending_a_durable_message.cs

```diff
@@ -0,0 +1,124 @@
+                .Done(c => c.WasCalled)
+                .Run();
+
+            Assert.True(context.WasSentDurable);
+        }
+
+        public class Context : ScenarioContext
+        {
+            public bool WasCalled { get; set; }
+            public bool WasSentDurable { get; set; }
+        }
+
+        public class Endpoint : EndpointConfigurationBuilder
+        {
+            public Endpoint()
+            {
+                EndpointSetup<DefaultServer>(
+                    b =>
+                    {
+                        b.GetSettings().Set("DisableOutboxTransportCheck", true);
+                        b.EnableOutbox();
+                    });
+            }
+
+            public class MyMessageHandler : IHandleMessages<MyDurableMessage>
+            {
+                public Context Context { get; set; }
+
+                public IBus Bus { get; set; }
+
+                public void Handle(MyDurableMessage message)
+                {
+                    Context.WasCalled = true;
+                }
+            }
+
+            public class StartMessageHandler : IHandleMessages<StartMessage>
+            {
+                public Context Context { get; set; }
+
+                public IBus Bus { get; set; }
+
+                public void Handle(StartMessage message)
+                {
+                    Bus.SendLocal(new MyDurableMessage());
+                }
+            }
+
+            public class DispatcherInterceptor : Feature
+            {
+                public DispatcherInterceptor()
+                {
+                    EnableByDefault();
+                    DependsOn<MsmqTransportConfigurator>();
+                }
+
+                protected override void Setup(FeatureConfigurationContext context)
+                {
+                    var originalDispatcher = EndpointConfiguration.builder.Build<ISendMessages>();
+                    var ctx = EndpointConfiguration.builder.Build<Context>();
+                    context.Container.ConfigureComponent(() => new SenderWrapper(originalDispatcher, ctx), DependencyLifecycle.SingleInstance);
+                }
+            }
+
+            public class EndpointConfiguration : IWantToRunBeforeConfigurationIsFinalized
+            {
+                public static IBuilder builder;
+
+                public void Run(Configure config)
+                {
+                    builder = config.Builder;
+                }
+            }
+
+
+            class SenderWrapper : ISendMessages
+            {
+                public SenderWrapper(ISendMessages wrappedSender, Context context)
+                {
+                    this.wrappedSender = wrappedSender;
+                    this.context = context;
+                }
+
+                public void Send(TransportMessage message, SendOptions sendOptions)
+                {
+                    context.WasSentDurable = message.Recoverable;
```

Comment:
sidenote: this method will be invoked twice, as the inital send is also wrapped by this class. Therefore the WasSentDurable property gets overwritten. As currently the value we're interested in, is the last write, that's fine though.